contracts/TalentLayerID.sol

```diff
@@ -216,7 +230,7 @@ contract TalentLayerID is ERC2771RecipientUpgradeable, ERC721Upgradeable, UUPSUp
     function freeMint(
         uint256 _platformId,
         address _userAddress,
-        string memory _handle
+        string calldata _handle
```

Comment:
@0xRomain I understand using calldata here saves gas, but the code would not even compile if I used "memory".
It looks like when converting data from string to bytes, the variable can't be in memory.